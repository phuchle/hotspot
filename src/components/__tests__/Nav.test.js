import React from 'react';
import renderer from 'react-test-renderer';
import Navigation from '../Nav';
import { MemoryRouter } from 'react-router-dom';

describe ('Navigation bar', () => {
  test('renders without crashing', () => {
    const component = renderer.create(
      <MemoryRouter>
        <Navigation />
      </MemoryRouter>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
