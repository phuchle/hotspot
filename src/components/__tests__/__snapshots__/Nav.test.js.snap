// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation bar renders without crashing 1`] = `
<nav
  className="navbar navbar-default"
  style={
    Object {
      "marginBottom": "0",
    }
  }
>
  <div
    className="container"
  >
    <div
      className="navbar-header"
    >
      <a
        className="navbar-brand"
        href="/"
        onClick={[Function]}
        style={
          Object {
            "color": "red",
          }
        }
      >
        HotSpot
      </a>
      <button
        className="navbar-toggle collapsed"
        onClick={[Function]}
        type="button"
      >
        <span
          className="sr-only"
        >
          Toggle navigation
        </span>
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
      </button>
    </div>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
    >
      <ul
        className="nav navbar-nav"
        role={null}
      >
        <li
          className=""
          role="presentation"
          style={undefined}
        >
          <a
            href="https://github.com/phuchle/hotspot"
            onClick={[Function]}
          >
            Source Code
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
